src/analyses/basePriv.ml

```diff
@@ -128,17 +128,15 @@ end
 
 (** Protection-Based Reading old implementation.
     Unsound!
-    Based on [sync].
-    Works for OSEK. *)
```

Comment:
I think we should aim to get rid of unsound privatizations!

Reply:
The question is if we want to keep `ProtectionBasedVesalPriv`: I don't think it's unsound, so maybe we can keep it.

Reply:
Vesal was also fine removing these old privatizations, so feel free to remove them (and a bunch of `Old` helper modules above too, if I remember correctly).

Reply:
Ok, I removed them now (hopefully with all helpers). Can you recheck that I didn't forget anything?

Reply:
Somehow I didn't see your comment about `ProtectionBasedVesalPriv` right before I commented, so I didn't even remember that. But since it's only point was to have a sound privatization that works with OSEK, there's no need for it anymore. It didn't go through the traces article rigor either, so there's no vouching for its performance or reliability.